src/zenml/zen_server/deploy/helm/templates/server-db-job.yaml

```diff
@@ -95,6 +181,27 @@ spec:
             - name: {{ $k }}
               value: {{ $v | quote }}
             {{- end }}
+            {{- if .Values.zenml.proxy.enabled }}
+            - name: HTTP_PROXY
+              value: {{ .Values.zenml.proxy.httpProxy | quote }}
+            - name: HTTPS_PROXY
+              value: {{ .Values.zenml.proxy.httpsProxy | quote }}
```

Comment:
for my understanding, these env variables are consumed by python libraries we use, automatically?

Reply:
exactly !